server/routes/discover.ts

```diff
@@ -704,4 +709,50 @@ discoverRoutes.get<{ language: string }, GenreSliderItem[]>(
   }
 );
 
+discoverRoutes.get<
+  { page?: number },
+  {
+    page: number;
+    totalPages: number;
+    totalResults: number;
+    results: WatchlistItem[];
+  }
+>('/watchlist', async (req, res) => {
```

Comment:
```suggestion
discoverRoutes.get<
  { page?: number },
  {
    page: number;
    totalPages: number;
    totalResults: number;
    results: WatchlistItem[];
  }
>('/watchlist', async (req, res) => {
```
Would be nice to add an optional `user` parameter, which would allow for displaying of watchlists on user profiles. (Would we want to add another permission to view other users' watchlists? Or restrict this only to the user themselves and admins?)

Reply:
This is also something we can maybe look at doing later as well.